etc/concourse/deploy-pipeline-vars.yml

```diff
@@ -121,6 +121,9 @@ jwtalgo: algo
 # Use templates true/false
 abacus-configure: true
 
+# Slack interval
+slack: 10m
```

Comment:
I think we should have bigger slack interval. The slack interval control several max delays:
* max downtime time for Abacus
* max delay of any resource provider
   * max time of app usage events for cf-bridge
   * max polling interval for some providers

Having the default to 10 minutes might be too restrictive for some use-cases. We usually set this to 3 or even 5 days, which is the time needed to fix a tough productive instance problem after a weekend, no matter if there is 24x7 or not :)